opennms-webapp/src/main/webapp/admin/index.jsp

```diff
@@ -176,6 +176,7 @@
             <%String status = "Unknown";
                 try {
                     NotifdConfigFactory.init();
+                    NotifdConfigFactory.getInstance().reload();
```

Comment:
is that possible to not call reload here ? (your init reload is removed, and postConstruct did not work properly?)